marketdata/rate/coinmarketcap/cmc_test.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func Test_normalizeRates(t *testing.T) {
+	provider := "cmc"
```

Comment:
Is this string somewhere else? Maybe add as const?

Reply:
we only use this string here